tests/node_modules/bats-assert/CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+# Change Log
```

Comment:
Do all of these non-functional files need to be in here?
(Licence, readme, etc)

Reply:
Removed